topalign.c:96:5: warning: implicit declaration of function ‘toupper’
prospero.c:63:3: warning: implicit declaration of function ‘strcpy’

--- SRC-1.3/prospero.c
+++ SRC-1.3/prospero.c
@@ -26,6 +26,7 @@
 */
 
 #include<stdio.h>
+#include<string.h>
 #include<math.h>
 #include"cl.h"
 #include"seq_util.h"
--- SRC-1.3/topalign.c
+++ SRC-1.3/topalign.c
@@ -26,6 +26,7 @@
 */
 
 #include<stdio.h>
+#include<ctype.h>
 #include<math.h>
 #include"seq_util.h"
 #include"ariadne.h"
